processor/src/decoder/tests.rs

```diff
@@ -31,7 +31,7 @@ const EIGHT: Felt = Felt::new(8);
 
 const INIT_ADDR: Felt = ONE;
 const FMP_MIN: Felt = Felt::new(crate::FMP_MIN);
-const SYSCALL_FMP_MIN: Felt = Felt::new(crate::SYSCALL_FMP_MIN);
+const SYSCALL_FMP_MIN: Felt = Felt::new(crate::SYSCALL_FMP_MIN as u64);
```

Comment:
I tried to do so, but it turns out that there are quite a lot of places where we use these constants (mostly `FMP_MIN`) as a stack elements in tests, so I decided that it will be better just to use `Felt::new()` in some places and don't cast them to the `u64` in a large number of places in the tests